ably/rest/channel.py

```diff
@@ -56,38 +53,22 @@ def __init__(self, ably, name, options):
         else:
             self.__cipher = None
 
-    def _format_time_param(self, t):
-        try:
-            return '%d' % (calendar.timegm(t.utctimetuple()) * 1000)
-        except:
-            return '%s' % t
+
+    @property
+    def  name(self):
+        return self.__name  
 
     @catch_all
     def presence(self, params=None, timeout=None):
         """Returns the presence for this channel"""
-        params = params or {}
-        path = '/channels/%s/presence' % self.__name
-        return self.__ably._get(path, params=params, timeout=timeout).json()
-
+        return self.__presence
```

Comment:
I am not following this, a presence request should be requesting present members via REST, however it does not seem to do this?